src/ModelTraits/SpatieTranslatable/HasTranslations.php

```diff
@@ -65,10 +66,10 @@ public static function create(array $attributes = [])
             if ($model->isTranslatableAttribute($attribute)) { // the attribute is translatable
                 $model->setTranslation($attribute, $locale, $value);
             } else { // the attribute is NOT translatable
-                $model->{$attribute} = $value;
+                $non_translatable[$attribute] = $value;
             }
         }
-        $model->save();
+        $model->fill($non_translatable)->save();
```

Comment:
More a side comment but this should DO something apart from nothing if that returns a non-expected (probably `false` or `void/null`) value?